web3-react/src/libs/wallet-connect.ts

```diff
@@ -0,0 +1,24 @@
+import { initializeConnector } from '@web3-react/core'
+import { WalletConnect } from '@web3-react/walletconnect'
+
+import { Connection, ConnectionType, onConnectionError } from './connections'
+import { CHAIN_TO_URL_MAP } from './constants'
+
+export function buildWalletConnectConnector() {
```

Comment:
nit: in `web3-react` as well as `interface` we just define all of that on a module-level, without having a function such as `buildWalletConnectConnector` to call.

I would consider having all of that code on a module-level and just having a static export default walletConnectConection (or `export const walletConnectConncetion = {}` depending on the style you take in the codebase).

Same comment applies for other `buildX` functions.

Reply:
I don't dislike the idea! I kept it this way for it be clearer, given this is an example. I feel like the drives the point better for the users.